### ISSUE TITLE
nuka-carousel stops click events from hitting child elements if 'dragging' prop is set to false

### ISSUE THREAD

**Issue by JWroe** [FormidableLabs/nuka-carousel#421]
### Describe Your Environment
- nuka-carousel version? **4.4.0**
- React? **16.5.0**
- Browser? **Chrome  69.0.3497.100**

### Describe the Problem
As per the title. If dragging is true I can click my button. If dragging is false I can't click the button. Snippet is below. 
```
        <Carousel 
          dragging={false}
          withoutControls={true}
        >
          <button onClick={() => alert('I was clicked')} />
        </Carousel>
```

How do I work around this so that users can't drag but can click? 

Thanks,

Jamie

**Comment by jflayhart** [comment]
Yup... https://github.com/FormidableLabs/nuka-carousel/issues/399

**Comment by jflayhart** [comment]
@sarmeyer https://github.com/FormidableLabs/nuka-carousel/pull/423

**Comment by sarmeyer** [comment]
fixed in #423
